### PR TITLE
different way of component creation

### PR DESCRIPTION
**Pull request by amiteshhh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,9 +4,61 @@ import './App.css';
 function App() {
```
Comment (alionaSipchuk) [inline src/App.js:4]: it's not a good practice to define into a file more than 1 component,
please move component to a separate folder wherever needed

File: src/App.js [hunk 2]
```diff
@@ -4,9 +4,61 @@ import './App.css';
...
   return (
     <div className="App">
-      Hello world!
+      <MessageElement></MessageElement>
```
Comment (alionaSipchuk) [inline src/App.js:7]: component without children please use shortcut for component
`<MessageElement />` wherever needed

File: src/App.js [hunk 3]
```diff
@@ -4,9 +4,61 @@ import './App.css';
...
+        <h4 className="App-message-heading">Message created using <code>React.PureComponent:</code></h4>
+        <span className="App-message">Hello World</span>
+      </header>
+    );
+  }
+}
+
+/** Using `React.createElement` */
+class MessageElement extends React.Component {
+
```
Comment (alionaSipchuk) [inline src/App.js:51]: redundant whitespace

File: src/App.js [hunk 4]
```diff
@@ -4,9 +4,61 @@ import './App.css';
...
+  return (
+    <header className="App-message-container">
+      <h4 className="App-message-heading">Message created using <code>Functional component:</code></h4>
+      <span className="App-message">Hello World</span>
+    </header>
+  );
+}
+
+/** Using `React.Component` */
+class MessageComponent extends React.Component {
```
Comment (alionaSipchuk) [inline src/App.js:26]: let's get _Component/PureComponent/createElement_ from 'react' destructively

Comment (amiteshhh) [reply src/App.js]: I didn't get this point. Could you please elaborate it more.

Comment (alionaSipchuk) [reply src/App.js]: ok, for example: 
```
import React, { Component } from 'react';

class MessageComponent extends Component {....}
```

File: src/message/index.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export * from './default.component';
```
Comment (alionaSipchuk) [inline src/message/index.js:1]: it's a very bad pattern to named files with components with dot character.
Please rename all corresponding files just like component name inside this file(of course file name must be with capital letter).

_src/message/FunctionalComponent.js
src/message/PureComponent.js_
_....._

Comment (amiteshhh) [reply src/message/index.js:1]: Oh. Actually I followed the naming convention from angular. Will make the changes as per your feedback.

**Pull request merged**